config/config.go

```
@@ -9,19 +9,19 @@ import (
 )
 
 type Config struct {
-	Network       string         `yaml:"network"`
-	NetworkNodes  []string       `yaml:"network_nodes"`
-	LocalNode     string         `yaml:"local_node"`
-	Database      Database       `yaml:"database"`
-	AuthIDs       []string       `yaml:"auth_ids"`
-	DiscordBot    DiscordBot     `yaml:"discord"`
-	GRPC          GRPC           `yaml:"grpc"`
-	Wallet        Wallet         `yaml:"main_net_wallet"`
-	TestNetWallet Wallet         `yaml:"test_net_wallet"`
-	Logger        Logger         `yaml:"logger"`
-	HTTP          HTTP           `yaml:"http"`
-	Phoenix       PhoenixNetwork `yaml:"phoenix"`
-	Telegram      Telegram       `yaml:"telegram"`
+	Network      string         `yaml:"network"`
+	NetworkNodes []string       `yaml:"network_nodes"`
+	LocalNode    string         `yaml:"local_node"`
+	Database     Database       `yaml:"database"`
+	AuthIDs      []string       `yaml:"auth_ids"`
+	GRPC         GRPC           `yaml:"grpc"`
+	Wallet       Wallet         `yaml:"wallet"`
+	Logger       Logger         `yaml:"logger"`
+	HTTP         HTTP           `yaml:"http"`
+	Phoenix      PhoenixNetwork `yaml:"phoenix"`
+	DiscordBot   DiscordBot     `yaml:"discord"`
+	Telegram     Telegram       `yaml:"telegram"`
+	TargetMask   int            `yaml:"target_mask"`
```

This is confusing. It's better to be different that Command's target mask.
A Bot's instance should have a property to indicate its usage.
So, let's call it: `BotName`

In the `.env` file we will have a new parameter "BOT_NAME" (first line)

It can have 3 values: "Pagu", "Pagu-Testnet", "Pagu-Moderator" 
Names other than that should raise an error.

Now, we know what kind of Bot we are deploying.
Based on the name we can register proper commands:

```
switch botName {
case "Pagu":
if util.IsFlagSet(flags,  TargetMaskMainnet) {...}

case "Pagu-Testnet":
if util.IsFlagSet(flags,  TargetMaskTestnet) {...}

case "Pagu-Moderator": 
if util.IsFlagSet(flags,  TargetMaskModerator) {...}
}
```